### ISSUE TITLE
Spinner value changes in the wrong direction after Reset All (rounding error?)

### ISSUE THREAD

**Issue by phet-steele** [phetsims/gravity-force-lab-basics#40]
All of the following is specific to keyboard interaction. Doing any of these steps with a mouse does not cause an issue (except for step 2, you can use a mouse there).

1. Increment a mass spinner above its default value (3+ for m1, 5+ for m2).
2. Reset All
3. Navigate back to the same mass spinner you incremented.
4. Attempt to decrement the spinner.

Instead, the number will increase on the first down arrow key. Further down arrow presses will correctly decrement the spinner.

Note: This can also happen in the mirror direction. Decrementing in step 1 followed by an attempt to increment in step 4 is also broken.

Seen on macOS 10.13.2 Chrome. For phetsims/QA/issues/83.

<details>
<summary>Troubleshooting Information</summary>

URL: https://www.colorado.edu/physics/phet/dev/html/gravity-force-lab-basics/1.0.0-dev.13/phet/gravity-force-lab-basics_all_phet.html?accessibility
Version: 1.0.0-dev.13 2018-01-11 18:09:43 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36
Language: en-US
Window: 1918x1014
Pixel Ratio: 2/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 32 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 80)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {"assert":{"sha":"928741cf","branch":"master"},"axon":{"sha":"104f3dcf","branch":"master"},"brand":{"sha":"53cf227b","branch":"master"},"chipper":{"sha":"c52ae6cb","branch":"2.0"},"dot":{"sha":"ee4d0e25","branch":"master"},"gravity-force-lab-basics":{"sha":"a6770ce9","branch":"master"},"inverse-square-law-common":{"sha":"3a3cd736","branch":"master"},"joist":{"sha":"1188c469","branch":"master"},"kite":{"sha":"e46f990e","branch":"master"},"phet-core":{"sha":"88a51822","branch":"master"},"phet-io":{"sha":"e31dae9a","branch":"master"},"phet-io-website":{"sha":"7440f48b","branch":"master"},"phet-io-wrapper-classroom-activity":{"sha":"c84e3046","branch":"master"},"phet-io-wrapper-lab-book":{"sha":"d128352d","branch":"master"},"phet-io-wrapper-sonification":{"sha":"fe4c6314","branch":"master"},"phet-io-wrappers":{"sha":"f844c39e","branch":"chipper2.0"},"phetcommon":{"sha":"85d72626","branch":"master"},"query-string-machine":{"sha":"b8bdcc28","branch":"master"},"scenery":{"sha":"f72ac66f","branch":"master"},"scenery-phet":{"sha":"7f2ddfbf","branch":"master"},"sherpa":{"sha":"9ca67479","branch":"master"},"sun":{"sha":"97f5c0ae","branch":"master"},"tandem":{"sha":"b77a8102","branch":"master"}}
</details>

**Comment by jessegreenberg** [comment]
Verified in https://www.colorado.edu/physics/phet/dev/html/gravity-force-lab-basics/1.0.0-dev.13/phet/gravity-force-lab-basics_all_phet.html?accessibility, but this is no longer happening in master. I think that change fixed this bug as well. @phet-steele could you please verify?

**Comment by phet-steele** [comment]
I do not see this in master. Will wait for next dev!

**Comment by phet-steele** [comment]
Fixed.
